e2e-tests/monitoring/03-check-metrics.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: |
+    . ../functions
+    set -o errexit
+    set -o xtrace
+
+    get_metric_values node_boot_time_seconds ${NAMESPACE}-$(get_cluster_name)-mysql-0 admin:admin
```

Comment:
We compare here something?

Reply:
`get_metric_values` fails if non numeric response returned from the PMM server and test step fails.